.gitignore

```diff
@@ -1,3 +1,5 @@
+mussidae/data/time-range-sources/
```

Comment:
Why are you git-ignoring this when you're also checking stuff in from this directory?